workshops/food-and-beverage/optimized-distribution/README-Milestone3.md

```diff
@@ -0,0 +1,88 @@
+# Milestone 3 - Invite a Distributor to the Uni
+The Uni you launched in Milestone 1 provides an easy for way the Supplier to add new products.  But what about the Distributor, who will issue purchase orders and coordinate deliveries? Your next milestone in this workshop is to add a new node to the Uni, allowing a new participant (the Distributor) to interface with the existing participant (the Supplier).
+
+## Expand the Uni
+Now that you have an initial listing of products, it's time to expand our Uni to include a Distributor.  The Distributor will then be able to issue purchase orders for the Supplier's products and schedule deliveries of those products.
+
+### Invite a Participant
+Starting from the Vendia Share [web application](https://share.vendia.net/):
+
+* Select your Uni by name
+* Click the `Invite participant` button in the upper-right section of the Uni details view
+* Enter the email address you used to register with Vendia and click `Send uni invite`
+  * **Note:** Using the same Vendia user account for both nodes in the Uni allows each workshop attendee to work independently.  Stay tuned for a more real-world scenario in Milestone 5.
```

Comment:
Were you thinking of the demo of using cross-account sharing? That's in [Milestone 7](README-Milestone7.md).